app/views/index.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Chitter</title>
+  <link href='/css/application.css' rel='stylesheet' type='text/css'>
+</head>
+<body>
+  <div id='peeps'>
+  <br>
+  <br>
+  <form action='/peeps/new' method='get'>
+    <input type='submit' class='peep_button' value='NEW PEEP!'>
+  </form>
+  <br>
+  <br>
+  <% Peep.all(order: [:id.desc]).each do |peep| %>
```

Comment:
Direct communication between the view and the model - this should always go via the controller to ensure correct encapsulation. I would expect to see `@peeps.each` here instead of `Peep.all`.